src/NotesLoaderSM.cpp

```diff
@@ -1124,8 +1126,7 @@ bool SMLoader::LoadFromSimfile( const RString &sPath, Song &out, bool bFromCache
 	{
 		int iNumParams = msd.GetNumParams(i);
 		const MsdFile::value_t &sParams = msd.GetValue(i);
-		RString sValueName = sParams[0];
-		sValueName.MakeUpper();
+		string sValueName = sParams[0];
```

Comment:
Little gutsy to assume this will always be an uppercase string. Any reason for removing the call?